backend/src/apiserver/common/const.go

```diff
@@ -44,6 +44,7 @@ const (
 	RbacResourceTypeVisualizations = "visualizations"
 
 	RbacResourceVerbArchive   = "archive"
+	RbacResourceVerbUpdate    = "update"
```

Comment:
You should also add `update` to default roles: https://github.com/kubeflow/pipelines/blob/cfe59139be68705dadaa1a268f095923042b829c/manifests/kustomize/base/pipeline/multi-user/view-edit-roles.yaml#L48

Reply:
@Bobgy I added "update" in this section.